presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -114,8 +114,6 @@ PrestoExchangeSource::PrestoExchangeSource(
       sslContext_,
       [](size_t bufferBytes) {
         RECORD_METRIC_VALUE(kCounterHttpClientPrestoExchangeNumOnBody);
-        RECORD_HISTOGRAM_METRIC_VALUE(
-            kCounterHttpClientPrestoExchangeOnBodyBytes, bufferBytes);
```

Comment:
@karteekmurthys Can we remove usage of `kCounterHttpClientPrestoExchangeOnBodyBytes` also?

Reply:
Done.